Parsers/ASimRegistryEvent/test/SentinelOne_vimRegistryEvent_SchemaTest.csv

```diff
@@ -0,0 +1,21 @@
+﻿Result
+"(1) Warning: Missing recommended field [ActorUserId]"
```

Comment:
any id coming for user from source?

Reply:
No. For "Alerts" type of data from SentinelOne we are not getting userId for RegistryEvent Data.